npm/extensions.bzl

```diff
@@ -26,13 +26,22 @@ def _npm_extension_impl(module_ctx):
         fail("A minimum version of Bazel 6 required to use rules_js")
 
     for mod in module_ctx.modules:
+        replace_packages = {}
```

Comment:
Should this be computed outside the `module_ctx.modules` loop?

Reply:
probably, since module extensions have a global view over the build, in *theory* you could have a translate lock from module A and a replacement coming from module B.